core/arch/arm/plat-rockchip/platform_config.h

```diff
@@ -85,6 +86,35 @@
 #define FIREWALL_DDR_BASE	0xff534000
 #define FIREWALL_DDR_SIZE	SIZE_K(16)
 
+#elif defined(PLATFORM_FLAVOR_rk3588)
+
+#define GIC_BASE		0xfe600000
+#define GIC_SIZE		SIZE_K(64)
+#define GICC_BASE		0
+#define GICD_BASE		GIC_BASE
+#define GICR_BASE		(GIC_BASE + 0x80000)
+
+#define UART0_BASE		0xfd890000
+#define UART0_SIZE		SIZE_K(64)
+
+#define UART1_BASE		0xfeb40000
+#define UART1_SIZE		SIZE_K(64)
+
+#define UART2_BASE		0xfeb50000
+#define UART2_SIZE		SIZE_K(64)
+
+#define UART3_BASE		0xfeb60000
+#define UART3_SIZE		SIZE_K(64)
+
+#define SGRF_BASE		0xfd58c000
```

Comment:
SGRF in RK3588 means "Secure GRF" (only TZ can access), but this register block is not a secure one. This address is for SYS_GRF according to the memory map with details in section 6.5 of the TRM.
![image](https://github.com/user-attachments/assets/1a2d0ae7-59f9-407c-beaf-788cc9f442dc)